src/com/company/M10/Task3.java

```diff
@@ -0,0 +1,40 @@
+package com.company.M10;
+
+import java.util.Arrays;
+import java.util.Map;
+
+public class Task3 {
+    public static void main(String[] args) {
+//        String sentence = "the day is sunny the the\n" +
+//                "the sunny is is";
+//        Map<String, Integer> words = new HashMap<>();
+//        calculateRepetitions(words, sentence);
+
+//        String inputFileName = "file.txt";
+//        Map<String, Integer> words = new HashMap<>();
+//        try (BufferedReader reader = new BufferedReader(new FileReader(inputFileName))) {
+//            String line;
+//            while ((line = reader.readLine()) != null) {
```

Comment:
according the task we should calculate count words in whole file, not needed calculate its in every line.

Reply:
fixed